sw/device/examples/hello_usbdev/hello_usbdev.c

```diff
@@ -105,10 +119,11 @@ int main(int argc, char **argv) {
   spid_send("SPI!", 4);
 
   uint32_t gpio_state = 0;
+  bool say_hello = 1;
```

Comment:
Only use `true` and `false` to initialize booleans.

Reply:
fixed